address_book/repository.py

```diff
@@ -1,15 +1,119 @@
+from typing import Optional
+from sqlalchemy import or_
 from sqlalchemy.orm import Session
 
-from . import db, schemas
+from . import schemas, models
 
 
-def create_address(sesh: Session, address: schemas.AddressEntryCreate):
-    db_address = db.AddressEntry(**address.dict())
-    sesh.add(db_address)
-    sesh.commit()
-    sesh.refresh(db_address)
+# region Address
+
+
+def get_address(db: Session, address_id: int) -> Optional[models.AddressEntry]:
+    return (
+        db.query(models.AddressEntry)
```

Comment:
I want to say that I thought the databases class had a `get_one` function or something like that. It's nothing that needs to be changed right now though